static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/columnFields.tsx

```diff
@@ -7,9 +7,8 @@ import {QueryFieldValue} from 'sentry/utils/discover/fields';
 import ColumnEditCollection from 'sentry/views/eventsV2/table/columnEditCollection';
 import {generateFieldOptions} from 'sentry/views/eventsV2/utils';
 
-import {WidgetType} from '../types';
-
-import {DisplayType} from './utils';
+import {WidgetType} from '../../../types';
+import {DisplayType} from '../../utils';
```

Comment:
I noticed this too, but since updating the `displayType/visualization` can only be done inside of the `widgetBuilder`, I think we should keep it in the `widgetBuilder` and in the future remove it from other places